f/radios/acre2/acre2_clientInit.sqf

```diff
@@ -119,3 +125,96 @@ if(_typeOfUnit != "NIL") then {
 
   };
 };
+
+// ====================================================================================
+
+// ASSIGN DEFAULT CHANNELS TO RADIOS
+// Depending on the squad joined, each radio is assigned a default starting channel
+
+if(!f_radios_settings_acre2_disableRadios) then {
+
+	private ["_presetArray","_presetLRArray","_radioSR","_radioLR","_radioExtra","_hasSR","_hasLR","_hasExtra","_groupID","_groupIDSplit","_groupChannelIndex","_groupLRChannelIndex","_groupName"];
+	
+	waitUntil {uiSleep 0.1; [] call acre_api_fnc_isInitialized};
+
+	_presetArray = switch (side _unit) do {
+  		case blufor: {f_radios_settings_acre2_sr_groups_blufor};
+	  	case opfor: {f_radios_settings_acre2_sr_groups_opfor};
+	  	case independent: {f_radios_settings_acre2_sr_groups_indfor};
+	  	default {f_radios_settings_acre2_sr_groups_indfor};
+	};
+
+	_presetLRArray = switch (side _unit) do {
+		case blufor: {f_radios_settings_acre2_lr_groups_blufor};
+	  	case opfor: {f_radios_settings_acre2_lr_groups_opfor};
+	  	case independent: {f_radios_settings_acre2_lr_groups_indfor};
+		default {f_radios_settings_acre2_lr_groups_indfor};
+	};
+
+	_radioSR = [f_radios_settings_acre2_standardSHRadio] call acre_api_fnc_getRadioByType;
+	_radioLR = [f_radios_settings_acre2_standardLRRadio] call acre_api_fnc_getRadioByType;
+	_radioExtra = [f_radios_settings_acre2_extraRadio] call acre_api_fnc_getRadioByType;
+
+	_hasSR = ((!isNil "_radioSR") && {_radioSR != ""});
+	_hasLR = ((!isNil "_radioLR") && {_radioLR != ""});
+	_hasExtra = ((!isNil "_radioExtra") && {_radioExtra != ""});
+
+	_groupID = groupID (group _unit);
+	_groupIDSplit = [_groupID, " "] call bis_fnc_splitString;
+
+	_groupChannelIndex = -1;
+  	_groupLRChannelIndex = -1;
+
+  	if ((count _groupIDSplit) > 2) then {
+		_groupName = toUpper (_groupIDSplit select (count _groupIDSplit - 1));
```

Comment:
Should be set to: toUpper (_groupIDSplit select (count _groupIDSplit - 2)) instead of -1 as the default group layout is "NATO BSL -"